docs/pages/introduction/why-not-expo.mdx

```diff
@@ -3,108 +3,110 @@ title: Limitations
 sidebar_title: Limitations
 ---
 
-import { InlineCode } from '~/components/base/code';
 import { Collapsible } from '~/ui/components/Collapsible';
 
-Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in the decisions they make.
+Your success will be limited if you don't know the limitations of your tools. A good software engineer strives to understand the tradeoffs in their decisions.
 
-<Collapsible summary="Used Expo before? Here are some recent changes to the managed workflow you should be aware of…">
+## EAS
 
-With the release of Expo Application Services, the managed workflow has far fewer limitations, and can be used for any project no matter the complexity:
+**With the release of Expo Application Services (EAS)**, building your app with Expo has far fewer limitations, and it can be used for any project, no matter the complexity. With [EAS Build](/build/setup):
 
-- All libraries available to React Native apps are available to Expo managed workflow apps built with [EAS Build](/build/setup), but _some_ may require a [Prebuild Config Plugin](/guides/config-plugins/) to be added. The addition of config plugins means:
-  - Any 3rd party push notification service can support managed workflow apps.
-  - Native libraries that integrate with proprietary services can support managed workflow apps.
-- Apps built with EAS Build can be as small as you want since the entire Expo SDK is no longer bundled in by default.
-- Apps built with EAS Build can target children under 13 years old.
+<Collapsible summary="All libraries available to React Native apps are available in Expo">
 
-These changes mean Expo can fit many more use cases than before.
+All libraries available to React Native apps are available in Expo projects. Some libraries may require a [Prebuild Config Plugin](/guides/config-plugins/) to be added. The addition of config plugins means:
+
+- Any third-party push notification service, such as OneSignal, is supported.
```

Comment:
A lot of these, but in particular this one, feel like not a limitation at all, but a previous limitation we've addressed. I think the original intent of this doc was to be the "anti-pitch" that's often at the beginning of a tech talk, a way of building trust with the developer by being open about why this thing they're opinionated in favor of may not be right. for them. So, we may want to avoid saying any _specifically_ positive bullet points in order to avoid diluting the anti-pitch. (generalities like "EAS Build is compatible with all RN projects, **but...**" I think are fair game)